addons/product/models/product.py

```diff
@@ -472,7 +450,7 @@ def _name_get(d):
 
         # Prefetch the fields used by the `name_get`, so `browse` doesn't fetch other fields
         # Use `load=False` to not call `name_get` for the `product_tmpl_id`
-        self.sudo().read(['name', 'default_code', 'product_tmpl_id', 'attribute_value_ids', 'attribute_line_ids'], load=False)
+        self.sudo().read(['name', 'default_code', 'product_tmpl_id'], load=False)
```

Comment:
This is a good concern, but m2m are not prefetched, and reading a m2m doesn't prefetch other fields either.

Prefetch is done to `select * from table` instead of `select a, b from table` and then later `select c from table`, but o2m/m2m do `select id from other_table`.